Controllers/StatusController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using API.Models;
+using Microsoft.AspNetCore.Mvc;
+using API.Models.DTO.Administrator;
+using Microsoft.AspNetCore.Identity;
+using System.Collections.Generic;
+using API.Models.DTO;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Authorization;
+
+namespace API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class StatusController : ApiControllerBase
+    {
+        public StatusController(ApiContext context, UserManager<Employee> userManager) :
```

Comment:
We have many statuses (orderStatus, employeeStatus, etc.). It is unclear which status controller is this,

Reply:
This status controller can be used get for many status because these methods is really small.

Reply:
My method, that is inside, it looks like it deals with employee statuses only. I think it would be better to clarify the naming.

Reply:
If we do it then when we wil need any other status we will have every time create one more Controller with one method?